consensus/spos/bls/subroundEndRound.go

```diff
@@ -197,6 +198,30 @@ func (sr *subroundEndRound) doEndRoundJobByLeader() bool {
 		return false
 	}
 
+	currentMultiSigner := sr.MultiSigner()
+	err = currentMultiSigner.SetAggregatedSig(sig)
+	if err != nil {
+		log.Debug("doEndRoundJobByLeader.SetAggregatedSig", "error", err.Error())
+		return false
+	}
+
+	err = currentMultiSigner.Verify(sr.GetData(), bitmap)
+	if err != nil {
+		log.Debug("doEndRoundJobByLeader.Verify", "error", err.Error())
+
+		err = sr.verifyNodesOnAggSigVerificationFail(currentMultiSigner)
```

Comment:
no need to pass the multisigner, as this is already available in sr.Multisigner()
same for computeAggSigOnValidNodes

Reply:
thanks :+1:
done